js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -488,6 +488,16 @@ const KeyboardNavigationController = core.ViewController.inherit({
                 this._resetFocusedView();
                 this._closeEditCell();
             }
+
+            if(this._isLastValidCell(this._focusedCellPosition) && direction === 'next') {
+                const $cell = this._getCellElementFromTarget(eventTarget);
+                const interactiveElements = $cell.find(INTERACTIVE_ELEMENTS_SELECTOR);
+
+                if(!interactiveElements.length || interactiveElements.last().is(eventTarget) || $cell.is(eventTarget)) {
```

Comment:
Use **_getInteractiveElement** for getting the last interactive element in a cell.
```suggestion
                const $lastInteractiveElement = this._getInteractiveElement($cell, true);

                if(!isElementDefined($lastInteractiveElement) || $lastInteractiveElement.is(eventTarget) || $cell.is(eventTarget)) {
```